readinglist/resource.py

```diff
@@ -216,10 +220,12 @@ def _extract_filters(self, queryparams):
             param = param.strip()
             value = native_value(value)
 
-            if param == '_sort':
+            # Ignore specific fields
+            if param.startswith('_') and param not in ('_since', '_to'):
```

Comment:
Well it just skips specific fields (that starts with '_') except `_since` and `_to` that are to be handled in this method. (It basically skips ``_sort``, ``_limit`` and ``_token``)